### PR TITLE
SRCH-4293 Update required configuration for redis/sidekiq for sidekiq version 7.x

### PR DESCRIPTION
**Pull request by krbhavith**
## Summary
- Brief summary of the changes included in this PR
- Any additional information or context which may help the reviewer
 
### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review. If you cannot complete an item below, replace the checkbox with the ⚠️ `:warning:` emoji and explain why the step was not completed.
 
#### Functionality Checks
 
- [x] You have merged the latest changes from the target branch (usually `main`) into your branch.
 
- [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket.

- [x] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release - SRCH-####, SRCH-####, SRCH-####** matching the Jira ticket numbers in the release.
 
- [x] Automated checks pass. If Code Climate checks do not pass, explain reason for failures:
 
#### Process Checks

- [x] You have specified at least one "Reviewer".

### CODE REVIEW

File: config/initializers/sidekiq.rb [hunk 1]
```diff
@@ -7,9 +7,9 @@
 end
 
 Sidekiq.configure_server do |config|
-  config.redis = { url: sidekiq['url'], namespace: sidekiq['namespace'] }
```
Comment (stevenbarragan) [inline config/initializers/sidekiq.rb:10]: Do you think a migration is necessary for things still on the sidekiq namespace? e.g. running or scheduled jobs

**Review by stevenbarragan — CHANGES_REQUESTED** [review]
I'm curious about why are we removing the namespace? and about migrating things out of that namespace

File: config/initializers/sidekiq.rb (hunk 1 above)
Comment (krbhavith) [reply config/initializers/sidekiq.rb:10]: @stevenbarragan  Sidekiq 7.x requires to move away from namespace, please find the other details in [SRCH-4293](https://cm-jira.usa.gov/browse/SRCH-4293). Regarding the migration, existing sidekiq instance will be stopped and new instance will be pointed to 2 going forward, refer to [SRCH-4308 ](https://cm-jira.usa.gov/browse/SRCH-4308)

**Review by stevenbarragan — APPROVED** [review]
Don't forget to plan migration before deploying this change

**Pull request merged**